### ISSUE TITLE
On Mobile after clicking on menu item it does not close drawer

### ISSUE THREAD

**Issue by tichaonax** [rjwats/esp8266-react#311]
On Mobile after clicking on menu item it does not close drawer, you have to click outside the menu to close.
This fix appears to work for me
Added the following line 53 to the file  
```
\interface\src\components\layout\LayoutDrawer.tsx
```
 onClick={onClose} // close drawer after click

![image](https://user-images.githubusercontent.com/1031407/178603341-9ebddf84-7333-42d7-8188-6f2a7dc85c44.png)

**Comment by rjwats** [comment]
Thanks Tichaona

**Comment by proddy** [comment]
I couldn't reproduce?

**Comment by tichaonax** [comment]
The issue is only on mobile! I am on android but was able to duplicate also on chrome browser on my laptop on small screen or if you select mobile.

**Comment by tichaonax** [comment]
Interesting it works on yours just verified but not on mine, let me take a look at the library.

**Comment by tichaonax** [comment]
> I tried on mobile and in chrome, couldn't reproduce. I guess I'm using newer libraries. My online demo is on https://ems-esp.derbyshire.nl/ems-esp/dashboard/data

you have some cool stuff there, on the extended Network Connection is that in the latest framework or its something you extended on your own.

**Comment by proddy** [comment]
> > I tried on mobile and in chrome, couldn't reproduce. I guess I'm using newer libraries. My online demo is on https://ems-esp.derbyshire.nl/ems-esp/dashboard/data
> 
> you have some cool stuff there, on the extended Network Connection is that in the latest framework or its something you extended on your own.

no, its a bunch of custom things I've added over time very specific to my https://github.com/emsesp/EMS-ESP32 project. I started maintaining a list of the differences but stopped as it was getting too noisy. Some new features like logging, telnet/console service, syslog and API calls are the big ones. On Network some tweaks for wifi performance and adding Ethernet support. MQTT is different too as my project has to send out hundreds of messages a minute and performance and stability were important.
